### PR TITLE
Add class work Quarto files

### PR DESCRIPTION
**Pull request by juliasilge**
I've gotten the R versions of these files ready to go here.

@isabelizimm can you add in Python versions of each of these?

For the flexdashboard thing I have here, I think the best option would be a nice Quarto report that does monitoring tasks. You'll see this once you start working through these files, but the idea with that file is something that renders as is for folks to see.

### CODE REVIEW

File: class-work/04-monitor-report.qmd [hunk 1]
```diff
@@ -0,0 +1,100 @@
+---
```
Comment (isabelizimm) [inline class-work/04-monitor-report.qmd:1]: My first instinct was to make this a Jupyter notebook, but the rendering to report is sooo much nicer for a `qmd` over `ipynb`. The API endpoint iframe renders really nicely in Quarto, which was my main concern!

File: class-work/02-deploy-python.qmd [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+rsc_url = os.getenv("CO_RSC_URL")
+
+v = VetiverModel(__)
+board = ___
+vetiver_pin_write(board, v)
+```
+
+## Model Card ❤️
+
+Open the Model Card template in RStudio by choosing "File" ➡️ "New File" ➡️ "R Markdown" ➡️ "From Template" ➡️ "Vetiver Model Card".
```
Comment (juliasilge) [inline class-work/02-deploy-python.qmd:58]: @isabelizimm can you update this for the best way for Python folks to get the Model Card template? Is it just `vetiver.model_card(path = ".")`?

File: class-work/03-deploy-python.qmd [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+Change your `VetiverAPI` object so it looks like:
+
+```python
+vetiver_api = vetiver.VetiverAPI(v)
+vetiver_api.vetiver_post(get_month_names, "get_month_names")
+api = vetiver_api.app
+```
+
+After you add a new endpoint, run the API locally again.
```
[the remark this answers is not in the document]
Comment (juliasilge) [reply class-work/03-deploy-python.qmd:135]: I think this is for `02-deploy-python.qmd`, correct? Let's add it back in to show folks what happens when they add something different (i.e. the visual documentation is created based on this). The idea is what a Python person would do here:
https://posit-conf-2023.github.io/vetiver/slides/02-deploy.html#/your-turn-6

File: class-work/04-monitor-python.qmd [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    estimate = "preds"
+  )
+metrics_df
+```
+
+```{python}
+plot_metrics(metrics_df)
+```
+
+Read about how you can use pins for versioning metrics: <https://rstudio.github.io/vetiver-r/reference/vetiver_pin_metrics.html>
```
Comment (juliasilge) [inline class-work/04-monitor-python.qmd:59]:
```suggestion
Read about how you can use pins for versioning metrics: <https://rstudio.github.io/vetiver-python/stable/reference/pin_metrics.html>
```
Is this right?

File: class-work/04-monitor-report.qmd (hunk 1 above)
Comment (juliasilge) [reply class-work/04-monitor-report.qmd:1]: Nice! 🙌

File: class-work/04-monitor-report.qmd [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+title: Monitor your model
+---
```
Comment (juliasilge) [inline class-work/04-monitor-report.qmd:3]:
```suggestion
title: Monitor your model
execute:
  echo: false
---
```

Should we do this and take it out of the individual chunks?

**Review by juliasilge — COMMENTED** [review]
This is looking great! 🎉 

Can you take a look at a couple of questions/suggestions?

File: class-work/02-deploy-python.qmd (hunk 2 above)
Comment (isabelizimm) [reply class-work/02-deploy-python.qmd]: done!

File: class-work/03-deploy-python.qmd (hunk 3 above)
[the remark this answers is not in the document]
Comment (isabelizimm) [reply class-work/03-deploy-python.qmd:135]: yes, that is correct! updated 👍

**Pull request merged**